components/resolved_ts/src/resolver.rs

```diff
@@ -541,15 +544,19 @@ impl Resolver {
         }
     }
 
+    // This may be inaccurate: a large txn may have multiple locks, but we only
+    // track one. But it's just for monitoring and diagnosis.
```

Comment:
Dose it mean the `lock_count` of `TxnLocks` is always 1 for a large transaction, after it is tracked?

Reply:
No. The comment was misleading. I've modified it.
The inaccuracy comes from untracking large txn locks, because we do not know the ts when untracking a lock.

Reply:
> The inaccuracy comes from untracking large txn locks, because we do not know the ts when untracking a lock.

This could be added to the comments too.